### DISCUSSION TITLE
Is day-format configurable ?

### DISCUSSION THREAD

**Discussion by artemus717** [apache/skywalking#12192] [Q&A] [answered]
sw_records-all-${day-format}

Can we change day-format to what we want ? Such as YYYY-MM-DD ?

**Comment by wu-sheng** [comment, marked as answer]
No, we don't support that.

**Comment by Superskyyy** [comment]
It's not hard to tune it from the source code. Feel free to do so.

**Reply by artemus717** [reply]
We run xxl-job to mange index life circle. I find skywalking has native TTL config. So I just use native function in skywalking.
Thanks
